### ISSUE TITLE
Tabs - Position of active tab underline and focus outline differs between top positioned tabs

### ISSUE THREAD

**Issue by sianford** [Sage/carbon#3551]
### Current behaviour
When moving through large size top positioned tabs, the active tab green underline and gold focus outline position differs, being either on, above or below the bottom border line. This is more apparent in Safari browser but affects Chrome, Firefox and Edge too. See movie below.

https://user-images.githubusercontent.com/12828099/103792131-2f318880-503b-11eb-9d17-b4f668b4f4ba.mov

### Expected behaviour
The position of the active tab underline and focus outline should be the same for all tabs.

### Reproducible example
See behaviour in this codesandbox: https://codesandbox.io/s/musing-robinson-9clrf?file=/src/index.js

### Suggested solution(s)
Maintain the same position of the active tab underline and focus outline for all tabs.

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| carbon-react     | 54.6.0
| carbon-factory   |
| react-scripts    | 3.0.1
| React            | 16.8.6
| Browser          | Safari, Chrome, Firefox, Edge Chromium
| npm              | 6.14.8
| Operating System | macOS Catalina 10.15.7

**Comment by sianford** [comment]
FE-3479
